examples/external-mysql/run-tests.sh

```diff
@@ -12,31 +12,35 @@ source "${SCRIPTPATH}/../../.github/scripts/parse-versions.sh"
 # shellcheck source=/dev/null
 source "${TESTDIR}/common.sh"
 
-helm_install=(helm upgrade --install --create-namespace)
-ns=spire-system
-
 teardown() {
-  helm uninstall --namespace "${ns}" spire 2>/dev/null || true
-  kubectl delete ns "${ns}" 2>/dev/null || true
-
+  helm uninstall --namespace "spire-server" spire 2>/dev/null || true
   helm uninstall --namespace mysql mysql 2>/dev/null || true
-  kubectl delete ns mysql 2>/dev/null || true
+  kubectl delete ns spire-server 2>/dev/null || true
+  kubectl delete ns spire-system 2>/dev/null || true
 }
 
 trap 'trap - SIGTERM && teardown' SIGINT SIGTERM EXIT
 
-"${helm_install[@]}" mysql mysql --version "$VERSION_MYSQL" --repo "$HELM_REPO_MYSQL" \
-  --namespace mysql \
+kubectl create namespace spire-system --dry-run=client -o yaml | kubectl apply -f -
+kubectl label namespace spire-system pod-security.kubernetes.io/enforce=privileged || true
+kubectl create namespace spire-server --dry-run=client -o yaml | kubectl apply -f -
+kubectl label namespace spire-server pod-security.kubernetes.io/enforce=restricted || true
+
+helm upgrade --install --create-namespace mysql mysql --version "$VERSION_MYSQL" --repo "$HELM_REPO_MYSQL" \
+  --namespace spire-server \
   --values "${DEPS}/mysql.yaml" \
   --wait
 
-"${helm_install[@]}" --namespace "${ns}" --values "${SCRIPTPATH}/values.yaml" \
+helm upgrade --install --create-namespace --namespace "spire-server" \
```

Comment:
The name space is already created above, probably do we need the extra `--create-namespace` here?

Reply:
yeah, I can remove that.

Reply:
Done